spec/models/piece_spec.rb

```diff
@@ -50,4 +54,124 @@
       king.errors.clear
     end
   end
+
+  describe 'is_obstructed?' do
+    before :all do
+      # @user = FactoryGirl.create(:user)
+      # sign_in @user
+      @game = FactoryGirl.create(:game)
+      @game.pieces.delete
+      @white_queen = Queen.create(
+        x_position: 0,
+        y_position: 2,
+        color: true,
+        game_id: @game.id,
+        captured: false
```

Comment:
if you drop and re-add your database tables, you shouldn't need to specify this as it will default to false